### ISSUE TITLE
How many build-in Filters do we have in jsonType feature

### ISSUE THREAD

**Issue by parisqian-misa** [Codeception/Codeception#2438]
In the document, I could not find a list of build-in filters. I saw `url`, `date` and `empty` if-blocks in the code, are they filters?

One more thing, do we have anything like `json`/`object` filter, just like what we have with `array`, to detect whether the value part is a valid json object?

**Comment by DavertMik** [comment]
> In the document, I could not find a list of build-in filters. I saw url, date and empty if-blocks in the code, are they filters

yes, and yes, they should be documented better. I will work on that

> One more thing, do we have anything like json/object filter, just like what we have with array, to detect whether the value part is a valid json object?

right now `objetct` is the same as `array`. However, this may (should) be improved
